net/ServerSocket.hpp

```diff
@@ -134,10 +134,10 @@ class LocalServerSocket : public ServerSocket
         ServerSocket(Socket::Type::Unix, clientPoller, std::move(sockFactory))
     {
     }
-    ~LocalServerSocket();
+    ~LocalServerSocket() override;
 
-    virtual bool bind(Type, int) override { assert(false); return false; }
```

Comment:
I don't think having `virtual` hurts. If anything, it's easier to notice virtual functions. Having `override` of course *helps*, so that's fine.

Reply:
`override` is indeed [important](https://clang.llvm.org/extra/clang-tidy/checks/modernize/use-override.html), i.e. signature changes/spelling/.. Not using the (redundant) `virtual` in the `virtual override` is IMHO reasonable and more safe as one could miss the more important `override` specifier itself.